firebase-ai/src/main/kotlin/com/google/firebase/ai/Chat.kt

```diff
@@ -206,6 +198,67 @@ public class Chat(
     return sendMessageStream(content)
   }
 
+  private suspend fun automaticFunctionExecutingTransform(
+    transformer: FlowCollector<GenerateContentResponse>,
+    tempHistory: LinkedList<Content>,
```

Comment:
Is there a good reason for this to be specifically a LinkedList instead of a MutableList?